### PR TITLE
Update UG (Quickstart guide, Commands, param constraints, glossary)

### PR DESCRIPTION
**Pull request by NicolasCwy**
1. Amend `delete` and `view` command documentation to take into account deleting/viewing a guest or staff
2. Fix invalid command format for Quickstart guide
3. Add _staff_id_ for param constraints 
4. Add _staff_ and _guest_ to glossary

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -198,9 +214,9 @@ Action | Format, Examples
 --------|------------------
 **Add** | `add n/NAME pn/PASSPORT_NUMBER [p/PHONE_NUMBER] [r/ROOM_NUMBER]` <br> e.g., `add n/Bing Cheng pn/T0134568D p/99999999 r/69`
 **Edit** | `edit pn/<PASSPORT_NUMBER> <FIELD_NAME>/<NEW_FIELD_DETAILS>` <br> e.g., `edit pn/X12345678A p/99999999`
-**Delete** | `delete pn/<PASSPORT_NUMBER>`<br> e.g., `delete pn/XNOO19390`
+**Delete** | Guest: `delete pn/<PASSPORT_NUMBER>`<br>Staff: `delete sid/<STAFF_ID>`<br> e.g., `delete pn/XNOO19390`
 **List** | `list`
-**View** | `view pn/<PASSPORT_NUMBER>`<br> e.g., `view g/X12345678A`
+**View** | `view pn/<PASSPORT_NUMBER>``view sid/<STAFF_ID>`<br> e.g., `view g/X12345678A`
```
Comment (rgbpokka) [inline docs/UserGuide.md:219]: I believe there's a typo in the example here, it should be pn instead of g right?

**Review by rgbpokka — CHANGES_REQUESTED** [review]
Minor typo

File: docs/UserGuide.md (hunk 1 above)
Comment (NicolasCwy) [reply docs/UserGuide.md:219]: Good catch, amended

**Review by rgbpokka — APPROVED** [review]
LGTM

**Review by alyssa-savier — COMMENTED** [review]
Looks good!

**Review by alyssa-savier — APPROVED** [review]

**Pull request merged**